demo/decorators.ts

```diff
@@ -0,0 +1,20 @@
+export const classDecorator = (constructor: Function) => {
+  Object.seal(constructor);
+  Object.seal(constructor.prototype);
+};
+
+export const methodDecorator = (value: boolean) =>{
+  return (target: unknown, propertyKey: string, descriptor?: PropertyDescriptor) => {
+    if (descriptor) {
+      descriptor.enumerable = value;
+    }
+  };
+};
+
+export const accessorDecorator = (value: boolean) => {
+  return (target: unknown, propertyKey: string, descriptor?: PropertyDescriptor) => {
+    if (descriptor) {
+      descriptor.configurable = value;
+    }
```

Comment:
Should we also have a demo decorator using `@Annotation`?

Reply:
I added one, I'm not sure it's strictly necessary, but I don't think it hurts anything.